src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,22 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Application application = new Application();
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            application.printBall(lottery.getRandomBall());
+        }
+    }
+
+    private void printBall(Ball ball) {
```

Comment:
I'd say that for the Main class we can make an exception and make it static to avoid creating an `Application` class instance

Reply:
I'll keep that in mind.